pkgs/top-level/python-packages.nix

```
@@ -5095,6 +5095,8 @@ in {
     else
       callPackage ../development/python-modules/numpy { };
 
+  numpy_1_18_5 = callPackage ../development/python-modules/numpy/1.18.5.nix { }
```

Missing semicolon breaks the build.

```suggestion
  numpy_1_18_5 = callPackage ../development/python-modules/numpy/1.18.5.nix { };
```

---

Done